model_validation/2023_03_20_initial_public_sample_data.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>I'm just trying to understand what this is saying. So the <code>not_in_census</code> dataframe contains people who show up in 2029 taxes (1040s) but but do not show up in the 2030 census, correct? How should we interpret the dates you calculated? Does the date represent either 1) the approximate date at which the simulant was recorded by the 1040 observer, if that simulant was alive at that time, or 2) the time of death of the simulant if they died before the 1040 observer recorded them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yep that's exactly it! The fact that many of the dates are before 2030 indicates that many of the extra people are no longer alive.